rfcs/008-map-indexes.md

```diff
@@ -31,17 +31,17 @@ document are to be interpreted as described in
 
 ## Terminology
 
-`Sequence`: a 13 byte value formed by combining the current `Incarnation` of the database and the `Versionstamp` of the transaction. Sequences are monotonically increasing even when a database is relocated across FoundationDB clusters. See (RFC002)[LINK TBD] for a full explanation.
+`Sequence`: a 13-byte value formed by combining the current `Incarnation` of the database and the `Versionstamp` of the transaction. Sequences are monotonically increasing even when a database is relocated across FoundationDB clusters. See (RFC002)[LINK TBD] for a full explanation.
```

Comment:
Right. It was just copied from the original. Would probably be better to have links out to other RFCs for a centralized definition instead of repeating it everywhere.